doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md

```diff
@@ -168,36 +184,41 @@ The configuration should match the following Avro schema.
 }
 ```
 
-### Administration ###
-To create a Twitter owner verifier, use either Administration UI or [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
-The following image example illustrates how to create a Twitter owner verifier from Administration UI.
-                                   
+#### Administration
+
+To create a Twitter owner verifier, use  [Administration UI]({{root_url}}Glossary/#administration-ui) or [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
+If you want to use the Administration UI, choose the **Twitter verifier** type from the drop-down list on the **Add user verifier details** page.
+
 ![twitter verifier](Admin-ui/verifier-twitter.png "twitter verifier")
 
-## Custom owner verifier ##
-It is possible to implement and plug-in custom owner verifiers. You can find corresponding instructions on the 
-[Creating custom owner verifier]({{root_url}}Customization-guide/Kaa-Server/Development-environment-setup/#creating-custom-user-verifier) page.
+### Custom owner verifier
 
+You can implement plug in your custom verifiers.
```

Comment:
s/You can implement plug in your custom verifier/You can implement a custom, plug in verifier